core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/WorkflowComponent.scala

```diff
@@ -44,11 +45,17 @@ trait WorkflowComponent {
 
   import driver.api._
 
+  implicit lazy val workflowStatusColumnType: JdbcType[WorkflowStatus] with BaseTypedType[WorkflowStatus] =
```

Comment:
This and the changes on lines 28 and 58 are the crux of this PR, everything else is fallout